plugin.video.embycon/resources/lib/menu_functions.py

```diff
@@ -224,6 +224,57 @@ def showMovieAlphaList():
     xbmcplugin.endOfDirectory(int(sys.argv[1]))
 
 
+def showTvShowAlphaList():
+    log.debug("== ENTER: showTvShowAlphaList() ==")
+
+    settings = xbmcaddon.Addon()
+    server = downloadUtils.getServer()
+    if server is None:
+        return
+
+    collections = []
+
+    item_data = {}
+    item_data['title'] = "#"
+    item_data['media_type'] = "tvshows"
+    item_data['path'] = ('{server}/emby/Users/{userid}/Items' +
+                         '?Fields={field_filters}' +
+                         '&Recursive=true' +
+                         '&NameLessThan=A' +
+                         '&IncludeItemTypes=Series' +
+                         '&ImageTypeLimit=1' +
+                         '&SortBy=Name' +
+                         '&SortOrder=Ascending' +
+                         '&format=json')
+    collections.append(item_data)
+
+    group_movies = settings.getSetting('group_movies') == "true"
+    alphaList = ["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "Y", "Z"]
+
+    for alphaName in alphaList:
+        item_data = {}
+        item_data['title'] = alphaName
+        item_data['media_type'] = "tvshows"
+        item_data['path'] = ('{server}/emby/Users/{userid}/Items' +
+                             '?Fields={field_filters}' +
+                             '&Recursive=true' +
+                             '&NameStartsWith=' + alphaName +
+                             '&IncludeItemTypes=Series' +
+                             '&ImageTypeLimit=1' +
+                             '&SortBy=Name' +
+                             '&SortOrder=Ascending' +
+                             '&format=json')
+        collections.append(item_data)
+
+    for collection in collections:
+        url = (sys.argv[0] + "?url=" + urllib.quote(collection['path']) +
+               "&mode=GET_CONTENT&media_type=" + collection["media_type"])
+        log.debug("addMenuDirectoryItem: {0} ({1})", collection.get('title'), url)
```

Comment:
log spam

Reply:
only in debug mode

Reply:
there is a lot of places that logging could be reduced.
if there was an effort to reduce overall logging it might help but I have no time for that task in this release.